src/App.js

```diff
@@ -1,21 +1,30 @@
-function App() {
+import { useDispatch, useSelector } from "react-redux";
+
+import userFetch from "./store/user/asyncActions";
+
+const App = () => {
+  const currentUser = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
   return (
     <div className="App">
       <header className="App-header">
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>Redux check</h1>
+        <div>
+          <input
+            type="button"
+            value="Send"
+            onClick={() => dispatch(userFetch())}
```

Comment:
it's better to create a separated callbacks for function props without writing them here